### PR TITLE
fix(react): add assertion function to assert condition

### PR DESCRIPTION
**Pull request by manudeli**
# assertion function to assert condition

- assert

## PR Checklist

- [x] I have written documents and tests, if needed.

### CODE REVIEW

File: packages/react/src/utils/assert.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+export function assert(value: boolean, message: string): asserts value {
+  if (!value) {
+    throw new Error(message)
+  }
+}
```
[the remark this answers is not in the document]
Comment (manudeli) [reply packages/react/src/utils/assert.ts:5]: I absolutely agree with your opinion. Thanks!👍

File: packages/react/src/utils/assert.spec.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      throw new Error('no type')
+    }
+  }
+}
+
+describe('assert', () => {
+  it('should assert condition', () => {
+    const data = get('Helper')
+    assert(data.type === 'Helper', "data.type should be 'Helper'")
+    expect(data.library).toBe('Suspensive')
```
[the remark this answers is not in the document]
Comment (manudeli) [reply packages/react/src/utils/assert.spec.ts:25]: I thought how to improve this during some minutes, but I couldn't imagine proper code to meet your expectation.
Could you suggest it as code? I need your help.

**Review by okinawaa — APPROVED** [review]
thx!

**Comment by manudeli** [comment]
> thx!

So much better spec code. Thanks for your review

**Pull request merged**